### ISSUE TITLE
Implement SA1133 (DoNotCombineAttributes)

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#805]
**Category:** Readability
**Title:** Do not combine attributes
**Description:** C# allows attributes to be combined, such as the following:

```csharp
[Export, Shared]
```

This form makes it difficult to distinguish which attributes are applied to a code element, and which arguments are passed to those attributes. To avoid this situation, only place one attribute within a single set of brackets, such as the following:

```csharp
[Export]
[Shared]
```

**Note:** A rule to ensure only one attribute is placed on a given line will be proposed separately.

**Comment by Przemyslaw-W** [comment]
+1

**Comment by Maxwe11** [comment]
:+1:

**Comment by vweijsters** [comment]
:+1:
